web/src/components/organisms/Project/Content/ContentList/index.tsx

```diff
@@ -23,9 +23,14 @@ const ContentList: React.FC = () => {
     [navigate, workspaceId, projectId],
   );
 
+  const handleItemAdd = useCallback(() => {
+    navigate(`/workspaces/${workspaceId}/${projectId}/content/${modelId}/details`);
+  }, [navigate, workspaceId, projectId, modelId]);
+
```

Comment:
This doesn't handle adding an item. Its a redirect. Same as above, please rename this more appropriately

Reply:
Renamed to `handleNavigateToItemForm`.